experiments/compositions/expanders/cel-expander/pkg/cel/engine.go

```diff
@@ -0,0 +1,59 @@
+package cel
+
+import (
+	"github.com/google/cel-go/cel"
+	"github.com/google/cel-go/common/types/ref"
+	"github.com/wzshiming/easycel"
```

Comment:
No objections to starting here, but we should probably replace this library with something more kubernetes